### ISSUE TITLE
Crashing upon server start. (1.16.5)

### ISSUE THREAD

**Issue by mcchouse** [jediminer543/JMT-MCMT#47]
I was trying to start a 1.16.5 modded server and it worked fine during startup, but as soon as it started the console filled up with errors and it crashed after a little while. I'm using the latest version of MCMT on forge 35.1.37 
heres the logs: [latest.log](https://github.com/jediminer543/JMT-MCMT/files/6047204/latest.log)

**Comment by KubiRemPL** [comment]
Can you use java 8?

**Comment by mcchouse** [comment]
do you mean JDK 8? Mine seems to be on JDK 11 right now. Do i need to downgrade?

**Comment by KubiRemPL** [comment]
Yeah, you should downgrade to JDK 8. For me it's much better, cuz it has CMS gc. JDK 8 should help with this Nashorn thing

**Comment by mcchouse** [comment]
Thanks a lot! that fixed the issue. Server runs fine now :D

**Comment by KubiRemPL** [comment]
No problem. Have a very performant server.
If you want to talk or whatever you can add me on discord KubiRemPL#6064

Best Regards
Kubi
